### DISCUSSION TITLE
Is there WriteableBitmap like things?

### DISCUSSION THREAD

**Discussion by haruby511** [AvaloniaUI/Avalonia#5549] [Q&A] [answered]
I'm planning a cross-platform desktop app.

The key feature what I need is dynamic bitmap.
In WPF, there is [WriteableBitmap](https://docs.microsoft.com/en-us/dotnet/api/system.windows.media.imaging.writeablebitmap?view=net-5.0).
It can edit pixel by pixel and can present by `Image` control.

I want to edit the bitmap from CPU than GPU. Therefore, I don't really need GL or DX bitmaps.
And I want that it be scaled or rotated by parent control.
Is there anything to fit?

**Comment by jp2masa** [comment, marked as answer]
There is `Avalonia.Media.Imaging.WriteableBitmap`.

**Reply by haruby511** [reply]
Thanks. I found it just now.
